SharkCage/CageManager/CageManager.cpp

```diff
@@ -123,13 +129,24 @@ std::wstring OnReceive(const std::wstring &message)
 	return path;
 }
 
+void LabelCage(CageLabeler * cage_labeler, FullWorkArea * full_work_area, HDESK *new_desktop)
```

Comment:
I don't think we need to pass the `CageLabeler` and `FullWorkArea` as params, local variables should be fine